src/main/java/eu/europeana/apikey/config/SecurityConfig.java

```diff
@@ -16,13 +16,21 @@
 
     @Override
     protected void configure(HttpSecurity http) throws Exception {
-        http.requiresChannel().requestMatchers(r -> r.getHeader("X-Forwarded-Proto") != null)
+        http.requiresChannel().requestMatchers()
+//        http.requiresChannel().requestMatchers(r -> r.getHeader("X-Forwarded-Proto") != null)
```

Comment:
I wonder why we had this before? And why we commented this out? 
Is it because requests should come from Pro or Portal and thus have such a header? If so shouldn't we reinstate this?

Reply:
I need to look into this, it might indeed be what you say about the requests coming from the portal so that it doesn't work when testing locally